src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -183,7 +183,7 @@
       </FilesToPackage>
 
       <!-- Include source files for symbol packages. -->
-      <FilesToPackage Include="@(Compile->'%(FullPath)')">
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="'$(ExcludeSourceFiles)' != 'true'">
```

Comment:
Lets please reverse the default and make the name more specific like "IncludeSourceFilesFromPackages".

Reply:
So exclude source files by default, unless the consuming repo opts in?

Reply:
Actually you can default set it to true if UseSourceLink=false.

Reply:
If we are going to use the UseSourceLink property you can keep this as "ExcludeSourceFilesInPackages"

Reply:
So to be clear - set `ExcludeSourceFilesInPackages=$(UseSourceLink)`, then continue using the same logic that's already in this PR? Also, do you know if there's a common place that both `FrameworkTargeting.targets` and this file import? I can't find one (in buildtools), which would mean I'd have to set a default value for `ExcludeSourceFilesInPackages` in both places.